molgenis/molgenis-emx2
# feat: catalogue can use google analytics

What are the main changes you did:
- add option to configure google analytics 

See: https://github.com/orgs/molgenis/projects/15/views/1?sliceBy%5Bvalue%5D=Dev&pane=issue&itemId=79942482&issue=molgenis%7CGCC%7C648

how to test:
- is a accept / localhost context configure google analytics and check if analytics are sent then analytics is setup and user accepts the cookies. 

todo:
